frontend/simple/views/GroupDashboard.vue

```diff
@@ -0,0 +1,40 @@
+<script>
+export default {
+  name: 'GroupDashboard',
+  computed: {
+    group () {
+      return this.$store.getters.currentGroup
+    }
+  },
+  methods: {
+    invite (groupId) {
+      this.$store.commit('setCurrentGroupId', groupId)
+      this.$router.push({path: '/invite'})
+    }
+  }
+}
+</script>
+
+<template>
+  <section class="container section">
+    <p class='is-pulled-right'>
+      <a class="button is-primary" @click.prevent="invite(group.id)">
+        <span class="icon is-small"><i class='fa fa-plus'></i></span>
+        <span><i18n>Invite members</i18n></span>
+      </a>
+    </p>
+    <h1 class="title is-1">{{ group.groupName }}</h1>
+    <p>{{ group.sharedValues }}</p>
+    <h5 class="subtitle is-4"><i18n>Members</i18n></h5>
+    <ul>
+      <li v-for="member in group.members">{{ member }}</li>
+    </ul>
+  </section>
```

Comment:
OK, so this looks like it contains markup that should be within components.

I.e. if you can make this PR just be the top-level skeleton for inserting components, that will help us divide tasks on who is working on what component.

So, in other words, the markup that should be here is:

- The left column that shows the groups you're in
- The right column that contains the components

There shouldn't be a listing of group members, nor anything related to inviting, as those should be within a 1-line mention of the component itself, e.g. `<members-list></members-list>` etc.